addons/l10n_ch/migrations/11.2/pre-migrate.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+
+def migrate(cr, version):
+    cr.execute("SELECT res_id FROM ir_model_data WHERE module = 'l10n_ch' AND name='account_tax_report_line_chtax_solde_formula'")
+
+    expression_id = cr.fetchone()
```

Comment:
```suggestion
    expression_id, = cr.fetchone()
```
Be careful, as `cr.fetchone()` returns a tuple with one element per column. You can also specify the index if you prefer.